.github/workflows/Python.yml

```diff
@@ -114,8 +133,8 @@ jobs:
         s3_id: ${{ secrets.S3_ID }}
         s3_key: ${{ secrets.S3_KEY }}
         signing_pk: ${{ secrets.DUCKDB_EXTENSION_SIGNING_PK }}
-        run_tests: ${{ startsWith(github.ref, 'refs/tags/v') && 0 || 1  }}
-        run_autoload_tests: ${{ startsWith(github.ref, 'refs/tags/v') && 0 || 1  }}
+        run_tests: ${{ inputs.skip_tests == 'true' && 0 || 1  }}
```

Comment:
this ternary like thing is broken I think, this is actually part of what caused the whole 0.10.0 release issue. It only works if the truthy value evaluates to true: https://github.com/actions/runner/issues/409#issuecomment-752775072. Inverting the condition should fix it

Reply:
just to double check, should I change it to:
```
run_tests: ${{ inputs.skip_tests != 'true' && 1 || 0  }}
```
?

Reply:
jep I think that should do it

Reply:
Should be fixed by: https://github.com/duckdb/duckdb/pull/11156/commits/127b5ef5b794ca09fab5aebfed5a50bccd20f9ff